lib/timers.js

```diff
@@ -232,6 +231,12 @@ function listOnTimeout() {
   } else if (list === refedLists[msecs]) {
     delete refedLists[msecs];
   }
+
+  // do not close reused handler which is unrefed in setInterval
```

Comment:
This comment seems a bit too specific, even if it is correct. What about:

```
// Do not close the underlying handle if its ownership has changed (e.g it was unrefed in its callback).
```
?